### DISCUSSION TITLE
Please add ability to browse gists on mobile app

### DISCUSSION THREAD

**Discussion by ClintonLang** [community/community#79135] [Mobile] [answered]
Please add ability to browse gists (at least personal gists) on the mobile app

**Comment by stevepopovich** [comment, marked as answer]
Hi @ClintonLang thanks for writing in. I'll let the team know about your request and this is something we are considering for the future.

**Reply by ClintonLang** [reply]
Awesome.  Thanks Steve.  Looking forward to it.

**Comment by sscornelius** [comment]
DigitalstarWorld

**Reply by sscornelius** [reply]
hell no

**Comment by rairkz** [comment]
> Please add ability to browse gists (at least personal gists) on the mobile

I use this feature daily. But, I am unable to handle on app.

Waiting for fast solution...
